From 582aef606a9deb0c27d0509b8c7b5a3b9c111ffc Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Sun, 25 Apr 2021 03:17:39 +0200
Subject: [PATCH 375/446] arm64: dts: sun50i-a64: pinephone: Power off the
 touch controller in sleep

Otherwise it stays on and consumes quite a bit of power.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index cdc95e9c7..782b447a7 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -482,6 +482,7 @@ touchscreen@5d {
 		VDDIO-supply = <&reg_ldo_io0>;
 		touchscreen-size-x = <720>;
 		touchscreen-size-y = <1440>;
+		poweroff-in-suspend;
 	};
 };
 
-- 
2.31.1

